src/pages/Details.js

```diff
@@ -74,14 +80,17 @@ export default function Details({ match }) {
         </ImageWrapper>
         <Wrapper>
```

Comment:
please continue your convention to specify what the wrapper is for in the name, like you did with `<ImageWrapper>`